### ISSUE TITLE
can save playlist and collections to the server for the docker version

### ISSUE THREAD

**Issue by imjonyzhang** [4gray/iptvnator#305]
**Is your feature request related to a problem? Please describe.**
can save playlist and collations to the server for the docker version.

**Describe the solution you'd like**
so that i can watch iptv from different devices and use the same m3u file and same collections.

**Additional context**
i don't know anything about javascript project. but i know something about writing code, if you can tell me how you save the playlist for now version, i can try it myself. thank you for your great work. thank you!!!

**Comment by cachho** [comment]
Yes, please add an option to save the changes to the filesystem.

**Comment by imjonyzhang** [comment]
这是自动回复邮件。您好，您的来信已收到。

**Comment by alanmilinovic** [comment]
Can you please add this for docker version.

**Comment by imjonyzhang** [comment]
这是自动回复邮件。您好，您的来信已收到。
